central/processindicator/datastore/datastore_impl_test.go

```diff
@@ -461,6 +461,61 @@ func (suite *IndicatorDataStoreTestSuite) TestAllowsRemoveByPod() {
 	suite.NoError(err, "expected no error trying to write with permissions")
 }
 
+func (suite *IndicatorDataStoreTestSuite) TestIndicatorPruneBatch() {
+	numIndicators := 5000
+	suite.setupDataStoreNoPruning()
+
+	ids := suite.buildIDsToPrune(numIndicators)
+
+	// Try to remove indicators by id
+	indicatorCount, err := suite.datastore.PruneProcessIndicators(suite.hasWriteCtx, ids)
+	suite.NoError(err)
+	suite.Equal(numIndicators, indicatorCount)
+
+	// Prune under batch size
+	numIndicators = 30
+	suite.setupDataStoreNoPruning()
+
+	ids = suite.buildIDsToPrune(numIndicators)
+
+	// Try to remove indicators by id
+	indicatorCount, err = suite.datastore.PruneProcessIndicators(suite.hasWriteCtx, ids)
+	suite.NoError(err)
+	suite.Equal(numIndicators, indicatorCount)
+
+	// Prune over batch size
+	numIndicators = 5234
```

Comment:
Where is the number 5234 coming from?

Reply:
I made it up.  That batch size is 5000, I just wanted to test some number over that to make sure batching was working appropriately.

Reply:
Could you document this please? Random numbers scare me :fearful: